website/tools/build_docs.js

```diff
@@ -132,32 +121,55 @@ const generateFinalMarkdown = (title, text, entityMap) => {
     return prettier.format(header + text, prettierConfig);
 };
 
+async function getExamplesFromRepo() {
+    await fs.emptyDir(EXAMPLES_DIR_NAME);
+    process.chdir(EXAMPLES_DIR_NAME);
+    const { body } = await httpRequest({
+        url: EXAMPLES_REPO,
+        json: true,
+    });
+    const builtExamples = await buildExamples(body);
+    await addExamplesToSidebars(builtExamples);
+}
+
+async function buildExamples(exampleLinks) {
+    const examples = [];
+    for (const example of exampleLinks) {
+        const responseStream = await httpRequest({
+            url: example.download_url,
+            stream: true,
+        });
+        try {
+            const pipeline = promisify(stream.pipeline);
+            await pipeline(responseStream, fs.createWriteStream(example.name));
+            const exampleName = example.name.split('.')[0];
+            examples.push(`examples/${exampleName.replace(/_/g, '-')}`);
+            console.log(`Rendering example ${example.name}`);
+        } catch (err) {
+            throw err;
+        }
+    }
+    return examples;
+}
+
+async function addExamplesToSidebars(examples) {
+    sidebars.examples = { Examples: examples };
+    fs.writeFileSync(path.join(__dirname, '..', 'sidebars.json'), JSON.stringify(sidebars, null, 4));
+    console.log('Saving examples to sidebars.json');
```

Comment:
Same here, use `saved`, because the file is already written.

Reply:
Put it before too.